INSTALL.md

```diff
@@ -26,23 +32,15 @@ The steps to follow are:
    * It is also possible to generate a runner configuration with the above command, using the `--config` switch.
    * e.g. `circleci runner token create --config my-namespace/my-resource-class my-token > launch-agent-config.yaml`.
 
-#### Installation Tooling
-
-The installation process assumes that the following utilities are installed on the system:
-
-* curl
-* sha256sum (installed as part of `coreutils`)
-* systemd
-
-Installation also requires permissions to create a user, and create directories under `/opt`.
-
-#### Job Running Requirements
+### Job Running Requirements
 
 Running jobs requires that the following tools are available on the machine:
```

Comment:
oh, we should also recommend installing git perhaps? our go version works, but the real one is better